ios/Sources/MediaFeature/MediaFeature.swift

```diff
@@ -7,26 +7,24 @@ public struct MediaState: Equatable {
     // In order not to use any networks for searching feature,
     // `feedContents` is storage to search from & `searchedFeedContents` is searched result from `feedContents`
     public var feedContents: [FeedContent]
-    public var searchedFeedContents: [FeedContent]
-    var isSearchResultVisible: Bool
+    public var searchedFeedContents: [FeedContent]?
```

Comment:
I want to know why this should be optional 🙇

Reply:
To avoid the condition below.
```swift
searchText.isEmpty && searchedFeedContents.isEmpty && isSearchResultVisible
```
This condition seems impossible but it actually exists during the animation.

You may notice the issue by revert this change and run the app on a simulator that enabled slow animations.